### PR TITLE
JEN-1066 add ubuntu 'disco' to PS jenkinses

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

File: jenkins/param.yml [hunk 1]
```diff
@@ -170,6 +170,7 @@
           - ubuntu:artful
```
Comment (EvgeniyPatlan) [inline jenkins/param.yml]: we should remove artful as we don't support it

File: jenkins/pipeline.groovy [hunk 2]
```diff
@@ -29,7 +29,7 @@ pipeline {
             description: 'Tag/Branch for Percona-TokuBackup repository',
             name: 'TOKUBACKUP_BRANCH')
         choice(
-            choices: 'ubuntu:artful\ncentos:6\ncentos:7\ni386/centos:6\nubuntu:trusty\nubuntu:xenial\nubuntu:bionic\nubuntu:cosmic\ndebian:jessie\ndebian:stretch\nroboxes-rhel8',
+            choices: 'ubuntu:artful\ncentos:6\ncentos:7\ni386/centos:6\nubuntu:trusty\nubuntu:xenial\nubuntu:bionic\nubuntu:cosmic\nubuntu:disco\ndebian:jessie\ndebian:stretch\nroboxes-rhel8',
```
Comment (EvgeniyPatlan) [inline jenkins/pipeline.groovy]: the same about artful

File: jenkins/trunk.yml [hunk 3]
```diff
@@ -172,6 +172,7 @@
           - ubuntu:artful
```
Comment (EvgeniyPatlan) [inline jenkins/trunk.yml]: the same

File: jenkins/valgrind-param.yml [hunk 4]
```diff
@@ -168,6 +168,7 @@
           - ubuntu:artful
```
Comment (EvgeniyPatlan) [inline jenkins/valgrind-param.yml]: the same

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: jenkins/valgrind-param.yml (hunk 4 above)
Comment (hors) [reply jenkins/valgrind-param.yml]: done

File: jenkins/trunk.yml (hunk 3 above)
Comment (hors) [reply jenkins/trunk.yml]: done

File: jenkins/pipeline.groovy (hunk 2 above)
Comment (hors) [reply jenkins/pipeline.groovy]: done

File: jenkins/param.yml (hunk 1 above)
Comment (hors) [reply jenkins/param.yml]: done

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**